data/data_set.go

```diff
@@ -124,11 +126,19 @@ func (d *DataSetFilter) Parse(parser structure.ObjectParser) {
 	d.ClientName = parser.String("client.name")
 	d.Deleted = parser.Bool("deleted")
 	d.DeviceID = parser.String("deviceId")
+	d.State = parser.String("state")
+	d.DataSetType = parser.String("type")
```

Comment:
the filter is called [d|D]ataSetType anywhere in the code except in the query params where it is called type... A bit confusing (also because the "type" field also exists). 
```suggestion
	d.DataSetType = parser.String("dataSetType")
```
Swaggo code comments should also be updated